src/__tests__/render.js

```diff
@@ -0,0 +1,36 @@
+import {render} from '@testing-library/vue'
+import '@testing-library/jest-dom/extend-expect'
+
+test('returns baseElement which defaults to document.body', () => {
+  const {baseElement} = render({template: '<div />'})
+  expect(baseElement).toBe(document.body)
+})
+
+test('returns custom baseElement', () => {
+  const {baseElement} = render(
+    {
+      template: '<div />',
+    },
+    {
+      baseElement: document.createElement('blink'),
```

Comment:
I really like the new interface. But it is different from the react one which might be misleading. There a `container` is used to create an element and the `baseElement` just acts as pointer to that: https://testing-library.com/docs/react-testing-library/api#container

Reply:
Hm, I see. I kind of like our API better, but it's true that keeping things as simillar as possible is a must. Thanks for pointing it out!